lib/api/kuzzle.js

```diff
@@ -234,6 +237,29 @@ class Kuzzle extends EventEmitter {
       });
   }
 
+   /** Manage errors to be thrown
+   *
+   * @param {...args} ...args
+   * @throws {KuzzleError}
+   */
+
+  throw(...args) {
+
+    const [domain, subdomain, errorName] = args;
+    const placeholders = args.length > 3 ? args.slice(3) : ' ';
+    const name = `${domain}-${subdomain}-${errorName}`;
+    const error = _.get(errorCodes, `${domain}.subdomains.${subdomain}.errors.${errorName}`);
+    if (!error) {
+      throw new errors.InternalError(format('Internal Error : Cannot find error in config file. %s', ...placeholders), name, 0);
```

Comment:
You don't need to remind that this is an internal error, since the type of error is `InternalError`.

```suggestion
      throw new errors.InternalError(`Unknown error: ${name}`, name, 0);
```

Also... we discussed this, and I thought we agreed that in that case, we had to either make Kuzzle crash, or we had to hardcode the "unknown error" name and code to return it. 

Here, you're creating an InternalError error with an error name that we know is invalid. This is inconsistent.

Remember that these errors are then forwarded to users via our API.